src/include/processor/operator/shortest_path.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "binder/expression/expression.h"
+#include "common/vector/value_vector_utils.h"
+#include "expression_evaluator/base_evaluator.h"
+#include "processor/operator/base_shortest_path.h"
+#include "processor/operator/physical_operator.h"
+#include "processor/operator/source_operator.h"
+#include "storage/storage_structure/column.h"
+#include "storage/storage_structure/lists/lists.h"
+
+using namespace kuzu::evaluator;
+
+namespace kuzu {
+namespace processor {
+
+class ShortestPathAdjCol : public BaseShortestPath {
```

Comment:
Separate these two classes to two files.

Reply:
Done